playwright.config.ts

```diff
@@ -46,30 +46,30 @@ const config: PlaywrightTestConfig = {
   projects: [
     {
       name: 'chromium',
-      grepInvert: /@mobile/,
+      grepInvert: /(^(?!@desktop)$|(@tablet|@mobile))/,
```

Comment:
Effectively matches tests either with `@desktop`, or none of `@desktop`, `@tablet` or `@mobile`.